### PR TITLE
[MINOR][DOCS] Update runtime dynamic configuration command

### PR DESCRIPTION
**Pull request by FereX98**
### What changes were proposed in this pull request?
The runtime dynamic configuration command is modified.

### Why are the changes needed?
It seems that in local mode, Spark takes *spark.driver.extraJavaOptions* as Java options for the JVM instead of *spark.executor.extraJavaOptions*.

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ val sc = new SparkContext(new SparkConf())
 Then, you can supply configuration values at runtime:
 {% highlight bash %}
 ./bin/spark-submit --name "My app" --master local[4] --conf spark.eventLog.enabled=false
-  --conf "spark.executor.extraJavaOptions=-XX:+PrintGCDetails -XX:+PrintGCTimeStamps" myApp.jar
+  --conf "spark.driver.extraJavaOptions=-XX:+PrintGCDetails -XX:+PrintGCTimeStamps" myApp.jar
```
Comment (dongjoon-hyun) [inline docs/configuration.md]: Hi, @FereX98 . So, does this PR aims to match this with `--master local[4]`?

**Comment by dongjoon-hyun** [comment]
ok to test

File: docs/configuration.md (hunk 1 above)
Comment (FereX98) [reply docs/configuration.md]: Yeah I guess so

Comment (dongjoon-hyun) [reply docs/configuration.md]: Could you check the other example in this file?
For example, line 2725 seems to be the same with this example.
- https://github.com/apache/spark/blame/19ffc9096339bde87f774b92f16c2441adf5841c/docs/configuration.md#L2725

**Comment by kiszk** [comment]
ping @FereX98

File: docs/configuration.md (hunk 1 above)
Comment (srowen) [reply docs/configuration.md:93]: This is fine. It's also fine to say `--master your-master-here` or something to indicate this doesn't matter

**Pull request closed without merging**